opv_tasks/task/exportviewertask.py

```diff
@@ -0,0 +1,179 @@
+            )
+            path_edges = self.virtualtour_paths = self._client_requestor.make_all(ressources.PathEdge, filters=(
+                Filter("id_path_details")==path_details.id_path_details,
+                Filter("id_path_details_malette")==path_details.id_malette)
+            )
+
+            for path_node in path_nodes:
+                virtualtour_path_dict["path_details"]["path_nodes"].append({
+                    "id_path_node": path_node.id_path_node,
+                    "id_malette": path_node.id_malette,
+                    "id_panorama": path_node.id_panorama,
+                    "id_panorama_malette": path_node.id_panorama_malette,
+                    "id_path_details": path_details.id_path_details,
+                    "id_path_details_malette": path_details.id_malette,
+                    "disabled": path_node.disabled,
+                    "original_id_sensors": path_node.original_id_sensors,
+                    "original_id_malette": path_node.original_id_malette,
+                    "original_gps_pos": path_node.original_gps_pos,
+                    "original_minutes": path_node.original_minutes,
+                    "original_degrees": path_node.original_degrees,
+                    "reconstructed_id_sensors": path_node.reconstructed_id_sensors,
+                    "reconstructed_id_malette": path_node.reconstructed_id_malette,
+                    "reconstructed_gps_pos": path_node.reconstructed_gps_pos,
+                    "reconstructed_minutes": path_node.reconstructed_minutes,
+                    "reconstructed_degrees": path_node.reconstructed_degrees
+                })
+                self.panorama_to_add.append("{}-{}".format(path_node.id_panorama, path_node.id_panorama_malette))
+
+            for path_edge in path_edges:
+                virtualtour_path_dict["path_edges"]["{}-{}".format(path_edge.id_malette, path_edge.id_path_edge)] = {
+                    "id_path_edge": path_edge.id_path_edge,
+                    "id_malette": path_edge.id_malette,
+                    "source_id_path_node": path_edge.source_path_node.id_path_node,
+                    "source_id_path_node_malette": path_edge.source_path_node.id_malette,
+                    "dest_id_path_node": path_edge.dest_path_node.id_path_node,
+                    "dest_id_path_node_malette": path_edge.dest_path_node.id_malette,
+                    "id_path_details": path_edge.path_details.id_path_details,
+                    "id_path_details_malette": path_edge.path_details.id_malette,
+                    "source_target_yaw_dest": path_edge.source_target_yaw_dest,
+                    "source_yaw_dest": path_edge.source_yaw_dest,
+                }
+
+            self.virtualtour_dict["virtualtour_paths"].append(virtualtour_path_dict)
+        
+        with open(self.virtualtour_dir / "{}-{}.json".format(self.virtualtour.id_virtualtour, self.virtualtour.id_malette), "w") as virtualtour_file:
```

Comment:
It's id_malette first to be coherent.
```suggestion
        with open(self.virtualtour_dir / "{}-{}.json".format(self.virtualtour.id_malette, self.virtualtour.id_virtualtour), "w") as virtualtour_file:
```